Libraries/DirectMLX.h

```diff
@@ -2004,7 +2004,11 @@ namespace dml
     // ---------------------------------------------------------------------------------------------------------------
 
     // If `axes` is not specified, by default this reduces the entire tensor to single element.
-    inline Expression Reduce(Expression input, DML_REDUCE_FUNCTION function, Span<const uint32_t> axes = {})
+    inline Expression Reduce(
+        Expression input, 
+        DML_REDUCE_FUNCTION function, 
+        DML_TENSOR_DATA_TYPE targetDataType = DML_TENSOR_DATA_TYPE_UNKNOWN, 
```

Comment:
The convention used elsewhere in DMLX is to call this parameter `outputDataType`. (You'll need to rename the local variable on line 2046 to something else). Also, I'd put this as the very last parameter to avoid breaking existing callers who supply a custom `axes`.